Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -218,7 +218,7 @@
 - type: entity
   name : disabler bolt
   id: BulletDisabler
-  parent: BaseBullet
+  parent: [ BaseBullet, BaseBulletTrigger ]
```

yamllint

[error] too many spaces inside brackets

(brackets)